resources/SettingsTabComponents/SettingsPane.qml

```diff
@@ -314,7 +277,11 @@ Rectangle {
             anchors.fill: parent
 
             TextEdit {
-                text: selectedRowField(_fieldName)
+                text: {
+                    if (_fieldName == "description")
+                        return selectedRowField(_fieldName) + "\n\nNotes:\n" + selectedRowField("notes");
```

Comment:
Can `Notes:` be moved into the `Description:` row label?

![image](https://github.com/swift-nav/swift-toolbox/assets/183436/3616bfa5-7935-49fb-8ca4-8291c265d842)

e.g. `property string _title: "Description"` would become `property string _title: "Description\n\nNotes:"`

Reply:
yeah but dont think the notes would always be aligned with the text inside the area tho